Spokestack/WebRTCVAD.swift

```diff
@@ -31,15 +31,29 @@ private var frameBuffer: RingBuffer<Int16>!
 
 /// Swift wrapper for WebRTC's voice activity detector.
 @objc public class WebRTCVAD: NSObject, SpeechProcessor {
-
+    
+    /// <#Description#>
```

Comment:
Are these empty `<#Description#>` tags cues for a documentation generator?

Reply:
good catch, meant to fill those in